spec/brew/gem_spec.rb

```diff
@@ -0,0 +1,62 @@
+require 'spec_helper'
+
+RSpec.describe Brew::Gem, type: :aruba  do
+  subject(:aruba_cmd) { run_complete "#{brew_gem} #{command}" }
+
+  let(:help_message) { Regexp.new Regexp.quote(Brew::Gem::CLI.help_msg.lines.first) }
+
+  context "aruba environment" do
+    it "doesn't contain any Bundler or RVM stuff" do
+      cmd = run_complete "env"
+      output = cmd.output
+      expect(output).to_not match(/^BUNDLE_/)
+      expect(output).to_not match(/^GEM_/)
+      expect(output).to_not match(/^RUBY(OPT|LIB)/)
+    end
+  end
+
+  context "help" do
+    let(:command) { "help" }
+
+    it { is_expected.to have_output(help_message) }
+
+    it { is_expected.to be_successfully_executed }
+  end
+
+  context "no command" do
+    let(:command) { "" }
+
+    it { is_expected.to have_output(help_message) }
+
+    it { is_expected.to_not be_successfully_executed }
+  end
+
+  context "unknown command" do
+    let(:command) { "unknown" }
+
+    it { is_expected.to have_output(/unknown command: #{command}/) }
+
+    it { is_expected.to have_output(help_message) }
+
+    it { is_expected.to_not be_successfully_executed }
+  end
+
+  context "install/uninstall" do #, announce_stderr: true, announce_stdout: true do
```

Comment:
Ah, it's actually a debug comment. If you delete `do #` then it changes to print stdout/stderr for the containing specs.

Related, I think I'm going to make that whole block an integration spec anyway so it can be exclude by Travis builds where `brew` isn't present.